svc-systems/systems/storage.go

```diff
@@ -127,7 +127,9 @@ func (e *ExternalInterface) CreateVolume(req *systemsproto.VolumeRequest) respon
 		response := common.GeneralError(http.StatusBadRequest, response.PropertyUnknown, errorMessage, []interface{}{invalidProperties}, nil)
 		return response
 	}
-
+	fmt.Printf("Volume Data is %+v ", volume)
```

Comment:
do we need this print statement? or can we use loggers

Reply:
done